### PR TITLE
Flogo list

### PR DESCRIPTION
**Pull request by skothari-tibco**

### CODE REVIEW

File: example/plugin/main.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package walrus
+
+import (
+	"fmt"
+
+	"github.com/project-flogo/cli/common"
+	log "github.com/sirupsen/logrus"
```
Comment (fm-tibco) [inline example/plugin/main.go]: get rid of logrus, a fmt.prints will suffice for this example

File: util/flogo.go [hunk 2]
```diff
@@ -7,6 +7,7 @@ import (
 	"io/ioutil"
 	"log"
 	"os"
+	fpath "path"
```
Comment (fm-tibco) [inline util/flogo.go]: not sure why you need to alias this path, the fpath makes it more confusing

File: util/flogo.go [hunk 3]
```diff
@@ -207,39 +215,102 @@ func ReadContribDescriptor(descriptorFile string) (*FlogoContribDescriptor, erro
 	return descriptor, nil
 }
 
-func GetAllImports(path string) ([]string, error) {
+func ParseImportPath(path string) (string, string) {
 
-	var results []string
-	bytes, err := ioutil.ReadFile(path)
+	// If @ is specified split
```
Comment (fm-tibco) [inline util/flogo.go]: make sure you can handle the alternate import format `github.com/project-flogo/contrib@latest:/activity/log`

File: util/flogo.go (hunk 2 above)
Comment (skothari-tibco) [reply util/flogo.go]: The code uses "path" as arguments in some functions. I can change those arguments to something like module path and so on.

File: util/flogo.go (hunk 3 above)
Comment (skothari-tibco) [reply util/flogo.go]: Sure. Thanks

File: util/flogo.go (hunk 2 above)
Comment (skothari-tibco) [reply util/flogo.go]: Should I?

Comment (fm-tibco) [reply util/flogo.go]: yes, because fpath sounds like file path and it really isnt.

File: go.mod [hunk 4]
```diff
@@ -3,6 +3,7 @@ module github.com/project-flogo/cli
 require (
 	github.com/msoap/byline v1.1.1
 	github.com/project-flogo/core v0.9.0-alpha.6
+	github.com/sirupsen/logrus v1.4.0
```
Comment (fm-tibco) [inline go.mod]: remove logrus

File: go.mod [hunk 5]
```diff
@@ -3,6 +3,7 @@ module github.com/project-flogo/cli
 require (
 	github.com/msoap/byline v1.1.1
 	github.com/project-flogo/core v0.9.0-alpha.6
```
Comment (fm-tibco) [inline go.mod]: update to v0.9.0-beta.1

**Pull request merged**